web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -51,41 +30,82 @@
                   </div>
                 </template>
                 <span>
-                  Embargoed datasets are hidden from public access until a specific time period has
+                  Embargoed Dandisets are hidden from public access until a specific time period has
                   elapsed. Uploading data to the DANDI archive under embargo requires a relevant
                   NIH award number, and the data will be automatically published when the embargo
                   period expires.
                 </span>
               </v-tooltip>
             </template>
-          </v-checkbox>
-          <v-text-field
-            v-if="embargoed"
-            v-model="awardNumber"
-            label="Award number*"
-            hint="Provide an NIH award number for this embargoed dataset.
-                Note: this can be changed at any time and additional award
-                numbers can be added later."
-            persistent-hint
-            :counter="120"
-            :required="embargoed"
-            outlined
-            class="mt-4 shrink"
-            style="width: 20vw;"
-            :rules="awardNumberRules"
-          />
+          </v-switch>
+        </div>
+        <h1>Title</h1>
+        <div>
+          Provide a title for this Dandiset. The title will appear in search
+          results and at the top of the home page for this Dandiset, so make it
+          concise and descriptive.
         </div>
+        <v-text-field
+          v-model="name"
+          label="Title"
+          :counter="nameMaxLength"
+          required
+          outlined
+          dense
+          class="my-4"
+        />
+
+        <h1>Description</h1>
+        <div>
+          Provide a description for this Dandiset. This will appear prominently
+          under the title in the home page for this Dandiset.
+        </div>
+        <v-textarea
+          v-model="description"
+          label="Description"
+          :counter="descriptionMaxLength"
+          required
+          outlined
+          dense
+          class="my-4"
+        />
         <div v-if="!embargoed">
+          <h1>License</h1>
+          <div>
+            Select a license under which to share the contents of this Dandiset.
+            You can learn more about <a
+              href="https://www.dandiarchive.org/handbook/35_data_licenses/"
+              target="_blank" rel="noopener">licenses
+            for Dandisets</a>.
+          </div>
           <v-select
             v-model="license"
             :items="dandiLicenses"
-            label="License*"
+            label="License"
             class="my-4"
             outlined
             dense
           />
         </div>
-        <small class="float-right font-weight-bold">*indicates required field</small>
+        <div v-else>
+          <h1>NIH Award Number</h1>
```

Comment:
Same as above comment.
```suggestion
          <div class="text-h1">NIH Award Number</div>
```

Reply:
See f1cef20b8d9db90d0a24220dcbc721393749f8eb.